packages/ffe-buttons/less/base-button.less

```diff
@@ -0,0 +1,245 @@
+.ffe-button {
+    appearance: none;
+    border: 2px solid transparent;
+    border-radius: 4px;
+    box-shadow: 0 1px 1px 0 rgba(0, 0, 0, 0.2);
+    color: @ffe-white;
+    cursor: pointer;
+    display: block;
+    font-family: "MuseoSansRounded-500", arial, sans-serif;
+    .ffe-fontsize-button;
+    line-height: 24px;
+    overflow: hidden;
+    padding: 8px 30px;
+    position: relative;
+    text-align: center;
+    text-decoration: none;
+    transition: all @ffe-transition-duration @ffe-ease;
+    user-select: none;
+    width: 100%;
+
+    &:focus {
+        outline: none;
+    }
+
+    &:hover {
+        box-shadow: 0 1px 2px 1px rgba(0, 0, 0, 0.2);
+    }
+
+    &:active {
+        box-shadow: inset 0 1px 1px 0 rgba(0, 0, 0, 0.25);
+    }
+
+    &--action&--ghost {
+        &,
```

Comment:
Unintended, or should the default and focus state look the same?

Reply:
According to the [original implementation](https://github.com/SpareBank1/designsystem/blob/master/packages/ffe-buttons/less/action-button.less#L16-L21), this is how it should look.